README.md

```diff
@@ -191,3 +196,316 @@ Local development can be supported using Docker Compose.  Please be aware that P
 
 Templates for APIs, UIs and Metabase/Oracle can be used to kickstart or extend projects.  Please visit our collaborators' [NR Architecture Templates](https://github.com/bcgov/nr-arch-templates) repository for more information.
+
+## Getting Started
+
+Initial setup is intended to take four hours or less.  This depends greatly on intended complexity, features selected/excluded and outside cooperation.
+
+### Contents
+
+* Documentation:
+    * *.md
+* Workflows:
+    * Pull Request-based (.github/workflows/pr-open.yml)
+    * On Close (.github/workflows/pr-close.yml)
+    * Main Merge (.github/workflows/main.yml)
+* Hello World! starter application
+    * TypeScript source in src/
+    * One Jest test in test/
+    * JavaScript container in Dockerfile
+* Misc:
+    * nestjs
+    * eslint
+    * lint-staged
+
+Not included:
+
+* Repository secrets
+* Environment secrets
+* Issues
+* Pull requests
+* JavaScript (transpiled/created in dist/)
+
+### Prerequisites
+
+The following are required:
+
+* BC Government IDIR accounts for anyone submitting requests
+* GitHub accounts for all participating team members
+    * [Sign Up is free](https://github.com/signup)
+* Membership in the BCGov GitHub organization
+    * Provide GitHub IDs to [BCGov's Just Ask](https://just-ask.developer.gov.bc.ca/)
+* Project namespaces (pick one):
+    * OpenShift - [Register a New Project](https://registry.developer.gov.bc.ca/public-landing)
+    * Amazon Web Services - coming soon!
+
+
+### Consuming This Template
+
+![image](./.github/graphics/newRepo.png)
+
+Create a new repository using this repository as a template.
+* Select bcgov/nr-quickstart-typescript under Repository template
+* Check Codecov | Code Coverage to grant access
+* Jira cannot be unchecked (I try every time!)
+
+
+### Setting Up the GitHub Repository
+
+
+#### Pull Request Handling
+
+Squash merging is recommended for simplified histories ad ease of rollback.
+
+Cleaning up merged branches is recommended for your DevOps Specialist's fragile sanity.
+
+From GitHub:
+1. Select Settings (gear, top right) -> General (selected automatically)
+2. Scroll to Pull Requests
+    * `[check] Allow squash merging`
+    * `[check] Automatically delete head branches`
+
+
+### Closing Repo-Mountie Issues
+
+repo-mountie is a BCGov bot that likes to spam us.  Here are a few issues to expect.
+
+Lets use common phrasing
+ * Includes examples of inappropriate and preferred phrasing
+ * The default branch should be named `main`
+ * Close the issue
+
+Add missing topics
+ * Topics improve discoverability
+ * Directions will be included
+ * Close the issue
+
+
+### Action Secrets
+
+Action Secrets are consumed by workflows, including 3rd party Actions.  Please use Environment secrets for highly sensitive content.
+
+Manage Action Secrets from your Repo > Settings > Secrets > Actions.
+
+
+#### Required
+
+**GHTOKEN**
+
+* Default token, not viewable, common to all repositories
+* Variable: `{{ secrets.GHTOKEN }}`
```

Comment:
is this one same as `GITHUB_TOKEN`

Reply:
That's correct.  I cleaned up the token, but don't want to do much else.  The setup section should be getting overhauled this sprint.